pkg/client/unversioned/remotecommand/remotecommand.go

```diff
@@ -194,12 +194,19 @@ func (e *Streamer) doStream() error {
 		if err != nil {
 			return err
 		}
-		defer remoteStdin.Reset()
```

Comment:
actually, I'm surprised the remote connection was being held open with the deferred Reset() in place previously